### PR TITLE
Remove sdx-common clone in docker

### PR DESCRIPTION
**Pull request by emilioward**
This PR fixes the issue of a cryptography error when running docker build in sdx-sequence

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM onsdigital/flask-crypto
+FROM onsdigital/flask-crypto-queue
```
Comment (necrophonic) [inline Dockerfile]: Why change to the queue version? `sdx-sequence` shouldn't be needing queue functionality?

File: Dockerfile [hunk 2]
```diff
@@ -15,8 +15,7 @@ EXPOSE 5000
 RUN apt-get update -y
 RUN apt-get upgrade -y
 RUN apt-get install -yq git gcc make build-essential python3-dev python3-reportlab
-RUN git clone https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
+
 RUN pip3 install --no-cache-dir -r /app/requirements.txt
 
 ENTRYPOINT ./startup.sh
```
Comment (necrophonic) [inline Dockerfile]: Container doesn't start:
```
[2017-09-14 11:10:03 +0000] [11] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/arbiter.py", line 557, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/workers/base.py", line 136, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.4/dist-packages/gunicorn/util.py", line 357, in import_app
    __import__(module)
  File "/app/server.py", line 13, in <module>
    from sequences import sequence, batch_sequence, image_sequence, json_sequence
ImportError: No module named 'sequences'
[2017-09-14 11:10:03 +0000] [11] [INFO] Worker exiting (pid: 11)
[2017-09-14 11:10:03 +0000] [8] [INFO] Shutting down: Master
[2017-09-14 11:10:03 +0000] [8] [INFO] Reason: Worker failed to boot.
```

**Review by necrophonic — CHANGES_REQUESTED** [review]

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: Dockerfile (hunk 2 above)
Comment (emilioward) [reply Dockerfile]: Issue has now been addressed 👍

**Review by JamesGardiner — APPROVED** [review]

**Review by dcdarrell9 — APPROVED** [review]
LGTM 👍

**Review by necrophonic — APPROVED** [review]

**Pull request merged**